src/gui/editorwidgets/qgscolorwidgetwrapper.cpp

```diff
@@ -75,5 +75,15 @@ bool QgsColorWidgetWrapper::valid() const
 void QgsColorWidgetWrapper::setValue( const QVariant& value )
 {
   if ( mColorButton )
-    mColorButton->setColor( QColor( value.toString() ) );
+  {
+    if ( value.isNull() )
```

Comment:
Should be fixed in 94d88e65d847823dc8f94412ea25a5b91d472aa8 . Now color buttons correctly support null values, so you should be able to remove these changes.